public/editor/scripts/project-list.js

```diff
@@ -20,6 +20,15 @@ require.config({
 require(["jquery", "constants", "analytics", "moment"], function($, Constants, analytics, moment) {
   var projects = document.querySelectorAll("tr.bramble-user-project");
   var locale = $("html")[0].lang;
+  var isLocalStorageAvailable = !!(window.localStorage);
+  var favorites = [];
+  if(isLocalStorageAvailable && localStorage.getItem("project-favorites") !== null){
```

Comment:
I see what you're saying about it turning into `null`. I think instead, let's remove this second check and when assigning favorites, handle it there:
```js
favorites = JSON.parse(localStorage.getItem("project-favorites")) || [];
```